telemetry/definitions/commonDefinitions.json

```diff
@@ -850,6 +905,86 @@
             "description": "View the App Runner service logs (the logs produced by App Runner)"
         },
+        {
+            "name": "auth_gettingStartedPage",
+            "description": "Landed on the getting started page",
+            "metadata": [
+                { "type" : "source" },
+                { "type" : "numberOfConnections"}
+            ] 
+        },
+        {
+            "name": "auth_connectToAWS",
+            "description": "Called when user clicks on getting ConnectToAWS from any source. e.g. explorer, developer tools, etc",
+            "metadata": [
+                { "type": "source" },
+                { "type": "numberOfConnections"}
+            ]
+        },
+        {
+            "name": "auth_awsResources",
+            "description": "Called upon user interacting with View, Modify, and Deploy AWS Resources Tab",
+            "metadata": [
+                { "type": "profileNameStatus"},
+                { "type": "accessKeyStatus"},
+                { "type": "secretStatus"},
+                { "type": "iamIdentityCenterStartURL"}, 
+                { "type": "iamIdentityCenterRegion", "required": false },
+                { "type": "errorMsg", "required": false},
+                { "type": "result"}
+            ]
+        },
+        {
+            "name": "auth_codeWhisperer",
+            "description": "Called when user clicks AI-powered Code suggestions from Codewhisperer tab under getting started. Since there are two ways of loggining we want to know status of each available option",
+            "metadata": [
+                { "type": "builderIDStatus"},
+                { "type": "iamIdentityCenterStartURL"}, 
+                { "type": "iamIdentityCenterRegion", "required": false },
+                { "type": "errorMsg", "required": false},
+                { "type": "result"}
+            ]
+        },
+        {
+            "name": "auth_codeCatalyst",
+            "description": "Called when user clicks Launch CodeCatalyst Cloud-based Environments tab under getting started",
+            "metadata": [
+                { "type": "builderIDStatus"},
+                { "type": "result"}
+            ]
+        },
+        {
+            "name": "auth_signUpForFree", 
+            "description": "Called upon user clicking sign up for free in IAM Credentials",
+            "metadata": [
+                { "type": "result"}, // can we know if the user came back signing up successfully?
+                { "type": "source"}
+            ]
+        },
+        {
+            "name": "auth_addIAMUserCredentials",  // redundant
+            "description": "Called when user clicks add IAM User Credentials",
+            "metadata": [
+                { "type": "source"},  // do we wanna do status check for each input field here? 
+                { "type": "result"}
+            ]
+        },
+        {
+            "name": "auth_addIAMIdentityProfile", //redundant 
```

Comment:
How is this different from `aws_modifyCredentials` in use today?
Have we considered having the same metric but introduce differentiation between IAM Identity center and IAM credentials?

Reply:
Thanks. I have mentioned in doc to utilize the existing aws_modifyCredentials metric